src/main/java/com/contrastsecurity/http/SecurityCheckForm.java

```diff
@@ -60,4 +68,9 @@ public SecurityCheckForm(String applicationName, AgentType agentLanguage) {
         this.applicationName = applicationName;
         this.agentLanguage = agentLanguage;
     }
+
+    public SecurityCheckForm(String applicationId, Long jobStartTime) {
```

Comment:
If you want job_start_time to be required. Can you mark the old constructors as deprecated and add a constructor for (name,language,jobStartTime) as well as we can remove the old constructors later.

Reply:
If you do not want job_start_time to be a required parameter, make a note in the javadocs comment that it is needed for some job outcome policy configurations. If job_start_time isn't going to be required, then we don't need this constructor.

Reply:
What other usage are there for the sdk methods? I made that constructor so it could be nice and clean with just applicationId and jobStartTime, since it can be sent by jenkins every time. Is there a common use case for customers using the sdk method for security checks without being connected to jenkins already? Then I could see a stronger need to leave it optional.